lib/ansible/plugins/action/copy.py

```diff
@@ -216,7 +216,7 @@ def _ensure_invocation(self, result):
             if self._play_context.no_log:
                 result['invocation'] = "CENSORED: no_log is set"
             else:
-                result["invocation"] = self._task.args.copy()
+                result["invocation"] = dict(module_args=self._task.args.copy())
```

Comment:
Now that the above will be changed, you can do `result["invocation"]["module_args"] = self._task.args.copy()`

Reply:
@jborean93 No I can't, result["invocation"] doesn't exist at this time.

Reply:
It will if you re-add the line above, see https://github.com/ansible/ansible/pull/51582#discussion_r252909279

Reply:
Ok I understood. Let me do the change.